<!DOCTYPE html>
<html lang="Es">



    <head>


        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <title>Curso JS (JON)</title>
    </head>
    <body>
        <script>

        let Comentario = "/-------------Comentario-------------/";


                    /* Manejo ed Errores */

                    /* holaa; */

        try{
            console.log("En el try se agrega el codigo a evaluar");
            Holahhhhh;
            console.log("Segundo mensaje en el try");   
        }catch(error){
            console.log("Captura cualquier error surgido o lanzado en el Try");
            console.log(error);
        }finally{
            console.log("El bloque finally se ejecutara siempre al final de un bloque Try-catch");
        }
        
        console.log(Comentario);



        try {
            let numero = "Hola";

            if(isNaN(numero)){
                throw new Error("Esto no es numero");

            }

            console.log(numero * numero);
            
        } catch (error) {
            console.log(`Se produjo el siguiente Error: ${error}`);
        }





        </script>
    </body>
</html>